src/app/frontend/replicationcontroller/detail/actionbar.html

```diff
@@ -18,7 +18,7 @@
                    object-meta="$ctrl.details.objectMeta">
 </kd-actionbar-logs>
 
-<kd-scale-button resource-kind-name="[[Replication Controller|Label 'Replication Controller' which appears at the top of the scale dialog, opened from a replication controller details page.]]"
+<kd-scale-button resource-kind-name="[[ReplicationController|Label 'Replication Controller' which appears at the top of the scale dialog, opened from a replication controller details page.]]"
```

Comment:
@kenan435 Is this value used in calls to backend? It cannot be done like that, because what if somebody uses Japanese translations?